src/database/migrations/20191126082619-create-volunteer.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.createTable("volunteers", {
+      id: {
+        type: Sequelize.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+      },
+      first_name: {
+        type: Sequelize.STRING,
+        allowNull: true
+  
+      },
+      last_name :{
+        type: Sequelize.STRING,
+        allowNull: true
+      },
+      createdAt: Sequelize.DATE,
```

Comment:
This looks wrong. Shouldn't it be:

```javascript
createdAt: {
  type: Sequelize.DATE,
  allowNull: true
}
```

?

Also, the name should be consistent with what's defined above (snake_case instead of camelCase).